nginx/cernopendata.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
The certificate says "cod-test.org" and "CERN Opendata".  Can you please update with better values?

- WRT "cod-test.org", that domain may look too broken to people trying this out. If this is for local developments only, then what about "localhost" or "localhost.localdomain" or somesuch; otherwise something like "opendata-dev.cern.ch" would suit better.
- WRT "CERN Opendata", the usual name is "CERN Open Data".

Something like:

```
Country Name (2 letter code) [GB]: CH
State or Province Name (full name) [Berkshire]: Switzerland
Locality Name (eg, city) [Newbury]: Geneva
Organization Name (eg, company) [My Company Ltd]: CERN
Organizational Unit Name (eg, section) []: IT-CDA-DR
Common Name (eg, your name or your server's hostname) []: localhost
Email Address []: opendata-team@cern.ch
```